src/Chat/Plugin/Rebecca.php

```diff
@@ -40,10 +40,13 @@ private function getCountdown(): string
         $timeLeft = $this->getTimeUntilNextFriday();
 
         return sprintf(
-            'Only %d days, %d hours and %d minutes left until Rebeccaday, OMG!',
+            'Only %d %s, %d %s and %d %s left until Rebeccaday, OMG!',
             $timeLeft->days,
+            $timeLeft->days === 1 ? 'days' : 'day',
             $timeLeft->h,
-            $timeLeft->i
+            $timeLeft->h === 1 ? 'hours' : 'hour',
+            $timeLeft->i,
+            $timeLeft->i === 1 ? 'minutes' : 'minute'
```

Comment:
Somewhat my bad. But you just copy/pasted it :-)

It will now display 1 days and 2 day :P